src/tuner/nccl_ofi_model.c

```diff
@@ -35,11 +34,11 @@ float nccl_ofi_tuner_compute_cost(struct nccl_ofi_tuner_model_params *params, st
 	 * device. Costs associated with out-of-order completions that could
 	 * stall the pipeline should be captured here as well.
 	 */
-	net_lat = (proto == NCCL_PROTO_SIMPLE)
-		    ? params->net_lat + ofi_nccl_tuner_net_comp_overhead()
-		    : params->net_lat;
-
-	p2p_lat = nccl_nvlink_lat[algo][proto];
+	const float base_latency = *base_costs[func][algo][proto];
```

Comment:
Ok I admit, this one is bending my mind a little 😅

You're passing the table to this function as a pointer to a 3-d array. To get something out of the table, don't you need to dereference first and then index, i.e.,
```
(*base_costs)[func][algo][proto];
```
?